astrodb_utils/photometry.py

```diff
@@ -185,10 +188,10 @@ def ingest_photometry(
                 "Add it with add_publication function."
             )
             if raise_error:
-                logger.error(msg)
-                raise AstroDBError(msg)
+                logger.error(msg+str(e))
```

Comment:
does this just print out the error immediately after a message? super trivial, but might we add a line break or two in between?

Reply:
how would you do that? I struggle with adding line breaks to strings.